caniusepython3/__main__.py

```diff
@@ -51,6 +51,15 @@ def projects_from_requirements(requirements):
     return valid_reqs
 
 
+def req_has_file_link(req):
+    url = getattr(req, 'url', None)
+    if url and url.lower().startswith('file:'):
+        return True
+    link = getattr(req, 'link', None)
+    if link and getattr(link, 'scheme', '') == 'file':
+        return True
+
```

Comment:
Can you have it return False explicitly? Stylistically I don't like relying on the implicit `return None` semantics.